manifests/init.pp

```diff
@@ -20,11 +22,12 @@
         require => Package["nginx"],
     }
 
-    if $default_site_source {
+    if $default_site_source or $default_site_content {
```

Comment:
can we change this if to require only one of the two?

Reply:
If both are provided, Puppet should complain (a `file` resource can't have both a `source` and `content` field set).